### PR TITLE
Modernize TGeoMgrFromDdd

### PR DESCRIPTION
**Pull request by Dr15Jones**
#### PR description:

- consolidated into 1 file in plugins directory
- added consumes use ESGetToken
- added fillDescriptions

#### PR validation:

Code compiles.
This is a minor technical change which should have no effect on the results.

### CODE REVIEW

**Comment by Dr15Jones** [comment]
This is needed by #28223

**Comment by Dr15Jones** [comment]
please test

**Comment by Dr15Jones** [comment]
+1

File: Fireworks/Geometry/plugins/TGeoMgrFromDdd.cc [hunk 1]
```diff
@@ -46,16 +46,64 @@
...
+  TGeoVolume* createVolume(const std::string& iName, const DDSolid& iSolid, const DDMaterial& iMaterial);
+  TGeoMaterial* createMaterial(const DDMaterial& iMaterial);
+
+  // ---------- member data --------------------------------
+
+  const int m_level;
+  const bool m_verbose;
+  const bool m_fullname;
+  std::string m_TGeoName;
+  std::string m_TGeoTitle;
```
Comment (fabiocos) [inline Fireworks/Geometry/plugins/TGeoMgrFromDdd.cc]: @Dr15Jones trivial side remark: that is ok with me, in other PR reviews I have seen suggesting more to use trailing "_" for local variables as an extensive interpretation of our code rules (which in my view say a slightly different thing)

File: Fireworks/Geometry/plugins/TGeoMgrFromDdd.cc [hunk 2]
```diff
@@ -46,16 +46,64 @@
...
+
+  std::map<std::string, TGeoShape*> nameToShape_;
+  std::map<std::string, TGeoVolume*> nameToVolume_;
+  std::map<std::string, TGeoMaterial*> nameToMaterial_;
+  std::map<std::string, TGeoMedium*> nameToMedium_;
+
+  edm::ESGetToken<DDCompactView, IdealGeometryRecord> viewToken_;
+};
```
Comment (fabiocos) [inline Fireworks/Geometry/plugins/TGeoMgrFromDdd.cc]: as done here...

**Comment by fabiocos** [comment]
+1

**Pull request merged**